requirements.txt

```diff
@@ -1,2 +1,8 @@
 Django>=4.2,<4.3
-wagtail==5.0.1
+wagtail==5.1.1
+# Production dependencies.
+gunicorn>=21.2.0,<22.0.0
+psycopg[binary]>=3.1.10,<3.2.0
+whitenoise>=5.0,<5.1
+dj-database-url>=2.1.0,<3.0.0
+django-storages[s3]>=1.14.0,<2.0.0
```

Comment:
If we end up supporting addons like Django Debug Toolbar, a `dev-requirements.txt` could be useful.

Reply:
👍 Good point. I was hesitant because it’s lots more files, but it really is more realistic for real-world projects.